src/script/lua_api/l_client.cpp

```diff
@@ -59,10 +59,28 @@ int ModApiClient::l_display_chat_message(lua_State *L)
 	return 1;
 }
 
+// show_formspec(formspec)
+int ModApiClient::l_show_formspec(lua_State *L)
+{
+
+	const char *formname = luaL_checkstring(L, 1);
+	const char *formspec = luaL_checkstring(L, 2);
+
+	ClientEvent event;
+	event.type = CE_SHOW_LOCAL_FORMSPEC;
+	// pointer is required as event is a struct only!
+	// adding a std:string to a struct isn't possible
```

Comment:
it's totally possible, const char* is not possible. I don't know who does this comment but it's surely totally wrong